src/presentation/controllers/action-controller.ts

```diff
@@ -0,0 +1,37 @@
+import { BaseController, GetRequest, PutRequest, RestController } from './base-controller'
+import { Request, Response } from 'express'
+import { ActionService } from '../../business-logic/services/interfaces/action-service'
+import { Action } from '../../model/entities/action'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { ActionDto } from '../dtos/action-dto'
+
+@RestController('/actions')
+export class ActionController extends BaseController {
+
+  constructor(private readonly actionService: ActionService, private readonly httpErrorHandler: HttpErrorHandler) {
+    super()
+  }
+
+  @GetRequest()
+  public async getActions(_reg: Request, res: Response): Promise<void> {
```

Comment:
I'm so confused, i commented on this once before as well i think. Is this not supposed to be `req` rather than `reg`? It's the same in the method below.

Reply:
Just took a look at all the existing controllers. Seems to be spelled `reg` everywhere, which as Ralle points out is wrong.

Reply:
I have fixed it in the other controllers, and created PR for that (#16).